File: hache.client/src/app/core/models/HistorialCaja.ts


export interface HistorialCaja{
  iD_HistorialCaja?: number;
  iD_TurnoCaja: number;
  iD_Usuario: number;
  iD_Local: number;
  tipoMovimiento: string;
  monto: number;
  fecha: Date;
  nombreUsuario?: string;
  observacion?: string;
}

File: Hache.Server/Entities/HistorialCaja.cs
namespace Hache.Server.Entities
{
    public class HistorialCaja
    {
        public int ID_HistorialCaja { get; set; }

        public int ID_TurnoCaja { get; set; }

        public int ID_Usuario { get; set; }

        public int ID_Local { get; set; }

        public string TipoMovimiento { get; set; }

        public decimal Monto { get; set; }

        public DateTime Fecha { get; set; }

        public string? nombreUsuario { get; set; }

        public string? Observacion { get; set; }

    }
}
